### PR TITLE
Restructured the content

### PR DESCRIPTION
**Pull request by Mariangela**
## Purpose

- Restructured the order of the content and revamped it based on an offline discussion with Dushani.
- Fix for https://github.com/wso2/docs-apim/issues/678

### CODE REVIEW

File: en/docs/Learn/APIController/ci-cd-with-wso2-api-management.md [hunk 1]
```diff
@@ -20,13 +20,34 @@ different ways of addressing the problem. The above diagram depicts a generic so
...
 -   DevOps: Control the deployment process
 
-API Developers and Publishers work with a version control system, which acts as a single source of truth for the pipeline. 
+API Developers and Publishers work with a version control system, which acts as a single source of truth for the pipeline.   
+
+## API Publisher based CI/CD
+
 API Developers can use WSO2 API Manager's Publisher to create APIs. CI/CD for API Manager relies on a Version Control system that acts as a Single Source of Truth for the pipeline. After APIs are exported from one environment, promoting it to the other environment is done via the `apictl`. It is capable of handling 
-environment-related configurations and can promote the API seamlessly to other environments via a single command.  
+environment-related configurations and can promote the API seamlessly to other environments via a single command.
```
Comment (dushaniw) [inline en/docs/Learn/APIController/ci-cd-with-wso2-api-management.md]: Shall we change this as,

API Developers can use WSO2 API Manager's Publisher Portal to create an API in a lower environment. CI/CD for API Manager relies on a Version Control system that acts as a Single Source of Truth for the pipeline. So, after the API is exported from the lower environment, the exported API artifacts can be committed to a source code management repository and the promotion of the APIs to the upper environments can be done after running the tests in the lower environment. This process of promoting the API seamlessly to multiple environments can be automated via the `apictl` and well-configured CI/CD tools. `apictl` makes this more simpler by handling per environment-related configurations.

File: en/docs/Learn/APIController/ci-cd-with-wso2-api-management.md [hunk 2]
```diff
@@ -120,7 +142,31 @@ The **apictl** can export an API as an archive from a lower environment (i.e., d
...
+specifications and also have custom parameter files. This reduces human intervention and boosts productivity.
+- For example, when an organization depends on a microservices architecture, this method can be utilized to create an automated pipeline to move Swagger/OpenAPI specifications to upper environments.
+
+For more information on initializing an API Project using OpenAPI/Swagger Specification, see 
+[Initialize an API Project]({{base_path}}/Learn/APIController/importing-apis-via-dev-first-approach/#initialize-an-api-project).
+
+<a name="E"></a>
+### (E.) - Prepare an API project for CI/CD
 
 1.  Extract the content (API will be exported as an archive to the
```
Comment (dushaniw) [inline en/docs/Learn/APIController/ci-cd-with-wso2-api-management.md]: This step 1 is needed only if you followed, step C before. Add **`Rename it to SwaggerPetstore for easy reference.`** to the same step1.

Keep step2 first sentence as it is. Remove the sentence **`Rename it to `SwaggerPetstore` for easy reference.`**

File: en/docs/Learn/APIController/ci-cd-with-wso2-api-management.md (hunk 1 above)
Comment (Mariangela) [reply en/docs/Learn/APIController/ci-cd-with-wso2-api-management.md]: Fixed in https://github.com/wso2/docs-apim/pull/677/commits/eb59e1a0122cdb90be9152a172194ca6a78ca83c

File: en/docs/Learn/APIController/ci-cd-with-wso2-api-management.md (hunk 2 above)
Comment (Mariangela) [reply en/docs/Learn/APIController/ci-cd-with-wso2-api-management.md]: Fixed in https://github.com/wso2/docs-apim/pull/677/commits/eb59e1a0122cdb90be9152a172194ca6a78ca83c

**Pull request merged**